### DISCUSSION TITLE
how to change text direction?

### DISCUSSION THREAD

**Discussion by ostazmas** [freeplane/freeplane#1328] [General]
Hello dear,
Thanks for your efforts.  I want to make right to left mind maps. in Arabic language. but the text goes by default left to right.
how can I support the Arabic language? 
Thanks again

**Comment by choumanb** [comment]
Kindly try the attached sample file (done on version 1.11.5
NOTE: Unzip the file to get the .mm file (cause the upload here doesn't support extension .mm.

Regards, 
[Test2.zip](https://github.com/freeplane/freeplane/files/12322389/Test2.zip)
